src/core/Akka.Cluster.Tests.MultiNode/Routing/ClusterBroadcastRouter2266BugfixSpec.cs

```diff
@@ -0,0 +1,200 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ClusterBroadcastRouter2266BugfixSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Akka.Actor;
+using Akka.Cluster.TestKit;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.Routing;
+using FluentAssertions;
+
+namespace Akka.Cluster.Tests.MultiNode.Routing
+{
+    public class ClusterBroadcastGroupSpecConfig : MultiNodeConfig
```

Comment:
Multinode spec from @mrrd, based on the round-robin group clustered router MNTR spec. 

Verifies that the router has the correct number of routees and that ALL of them receive each message.